### PR TITLE
sg3_utils: bump to 1.44

### PR DESCRIPTION
**Pull request by grueni**

### CODE REVIEW

File: components/sysutils/sg3_utils/Makefile [hunk 1]
```diff
@@ -48,21 +47,23 @@ CFLAGS += -std=c99 $(CPP_XPG6MODE)
 CONFIGURE_OPTIONS +=	--disable-static
 CONFIGURE_OPTIONS += CPPFLAGS="$(CPPFLAGS)"
 CONFIGURE_OPTIONS += CFLAGS="$(CFLAGS)"
+CONFIGURE_OPTIONS += --bindir=$(USRBINDIR)
```
Comment (pyhalov) [inline components/sysutils/sg3_utils/Makefile]: This is confusing. We get `--bindir=/usr/bin/amd64 --bindir=/usr/bin` in configure args. 
1) Libs going to /usr/lib/amd64 is fine (and correct for 64-bit libraries). 
2) Use PREFERRED_BITS=64 like in https://github.com/OpenIndiana/oi-userland/blob/b538287cfdbce379cd40a703c72a76f4983d15fa/components/sysutils/findutils/Makefile#L25 to switch CONFIGURE_BINDIR.

File: components/sysutils/sg3_utils/manifests/sample-manifest.p5m [hunk 2]
```diff
@@ -79,72 +83,83 @@ file path=usr/bin/sg_write_buffer
...
 file path=usr/include/scsi/sg_lib.h
 file path=usr/include/scsi/sg_lib_data.h
+file path=usr/include/scsi/sg_pr2serr.h
 file path=usr/include/scsi/sg_pt.h
+file path=usr/include/scsi/sg_pt_nvme.h
+file path=usr/include/scsi/sg_unaligned.h
+link path=usr/lib/$(MACH64)/libsgutils2.so target=libsgutils2.so.2.0.0
+link path=usr/lib/$(MACH64)/libsgutils2.so.2 target=libsgutils2.so.2.0.0
+file path=usr/lib/$(MACH64)/libsgutils2.so.2.0.0
 link path=usr/lib/libsgutils2.so target=libsgutils2.so.2.0.0
```
Comment (pyhalov) [inline components/sysutils/sg3_utils/manifests/sample-manifest.p5m]: Where from do we get 32-bit libs here?

File: components/sysutils/sg3_utils/sg3_utils.p5m [hunk 3]
```diff
@@ -97,75 +102,87 @@ file path=usr/bin/sg_write_buffer
...
 file path=usr/include/scsi/sg_cmds_mmc.h
 file path=usr/include/scsi/sg_lib.h
 file path=usr/include/scsi/sg_lib_data.h
+file path=usr/include/scsi/sg_pr2serr.h
 file path=usr/include/scsi/sg_pt.h
+file path=usr/include/scsi/sg_pt_nvme.h
+file path=usr/include/scsi/sg_unaligned.h
+link path=usr/lib/$(MACH64)/libsgutils2.so target=libsgutils2.so.2.0.0
+link path=usr/lib/$(MACH64)/libsgutils2.so.2 target=libsgutils2.so.2.0.0
+file path=usr/lib/$(MACH64)/libsgutils2.so.2.0.0
```
Comment (pyhalov) [inline components/sysutils/sg3_utils/sg3_utils.p5m:116]: Where from do we get 32-bit libs here?

**Comment by grueni** [comment]
Thanks for the hint 'PREFERRED_BITS=64'.
I updated the PR accordingly.

File: components/sysutils/sg3_utils/sg3_utils.p5m [hunk 4]
```diff
@@ -23,8 +23,8 @@
 # Copyright (c) 2012, 2013, Oracle and/or its affiliates. All rights reserved.
 #
 
-# Read the section 1m from section 8.
-<transform file path=usr/share/man/man1m/(.+).1m$ -> set action.hash usr/share/man/man8/%<1>.8 >
+# prevents "64bit file in 32bit path" errors
```
Comment (pyhalov) [inline components/sysutils/sg3_utils/sg3_utils.p5m]: Please, remove this, it shouldn't be needed.

**Pull request merged**